googleapis/google-cloud-node
# migrate code from googleapis/nodejs-dms

- feat: initial release
- feat: add samples (#2)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#3)
- chore: release 1.0.0 (#1)
- chore(deps): update dependency gts to v3 (#6)
- chore(deps): update dependency sinon to v11 (#7)
- chore(deps): update dependency @types/node to v14 (#5)
- chore: update broken link in comments (#10)
- chore(nodejs): remove api-extractor dependencies (#13)
- docs: update link in comments (#15)
- build(node): fix package name hint (#19)
- fix(deps): google-gax v2.17.0 with mTLS (#22)
- chore: release 1.0.1 (#24)
- fix(deps): google-gax v2.17.1 (#25)
- chore: release 1.0.2 (#26)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#27)
- chore: release 1.0.3 (#28)
- chore(nodejs): update client ref docs link in metadata (#33)
- fix(build): migrate to using main branch (#35)
- chore: release 1.0.4 (#36)
- fix(deps): google-gax v2.24.1 (#37)
- chore: release 1.0.5 (#38)
- feat: turns on self-signed JWT feature flag (#39)
- chore: release 1.1.0 (#40)
- docs(samples): add auto-generated samples for Node with api short name in region tag (#46)
- docs: Fix NOT_STARTED state documentation in MigrationJob resource (#49)
- build(node): update deps used during postprocessing (#1243) (#50)
- chore: fix the wrong post processor image (#53)
- chore(deps): update dependency @types/node to v16 (#52)
- chore(cloud-rad): delete api-extractor config (#54)
- chore(deps): update dependency sinon to v12 (#55)
- docs(samples): add example tags to generated samples (#56)
- build: add generated samples to .eslintignore (#57)
- docs(node): support "stable"/"preview" release level (#1312) (#60)
- chore: add api_shortname and library_type to repo metadata (#59)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#62)
- test(nodejs): remove 15 add 16 (#1322) (#64)
- build: update copyright year to 2022 (#67)
- chore(deps): update dependency sinon to v13 (#71)
- docs(nodejs): version support policy edits (#1346) (#73)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#74)
- chore: update v2.14.2 gapic-generator-typescript (#78)
- build(node): update client library version in samples metadata (#1356) (#85)
- chore(deps): update dependency @types/mocha to v9 (#86)
- chore(deps): update dependency sinon to v14 (#89)
- build!: update library to use Node 12 (#90)
- chore(main): release 2.0.0 (#92)
- chore(deps): update dependency jsdoc-fresh to v2 (#95)
- chore(deps): update dependency jsdoc-region-tag to v2 (#96)
- feat: support regapic LRO (#97)
- chore(deps): update dependency linkinator to v4 (#100)
- chore(main): release 2.1.0 (#98)
- chore: remove unused proto imports (#101)
- fix: better support for fallback mode (#102)
- fix: change import long to require (#103)
- fix: do not import the whole google-gax from proto JS (#1553) (#106)
- fix: allow passing gax instance to client constructor (#107)
- chore(main): release 2.1.1 (#104)
- fix: preserve default values in x-goog-request-params header (#108)
- test: use fully qualified request type name in tests (#111)
- fix(deps): use google-gax v3.5.2 (#115)
- chore(deps): update dependency @types/node to v18 (#113)
- chore(deps): update dependency jsdoc to v4 (#117)
- Update .repo-metadata.json
- fix: regenerated protos JS and TS definitions (#118)
- chore(main): release 2.1.2 (#109)
- Update .repo-metadata.json
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
